src/Forms/Prism.Forms/Navigation/IInitializeAsync.cs

```diff
@@ -2,8 +2,16 @@
 
 namespace Prism.Navigation
 {
+    /// <summary>
+    /// Asynchronous initialization of navigation
+    /// </summary>
     public interface IInitializeAsync
     {
+        /// <summary>
+        /// Asynchronously initialize method which initializes the navigation.
```

Comment:
No need to add "asynchronously" in the description. Invoked when the View or ViewModel is first created during the navigation process.

Reply:
done